packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -550,7 +550,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 // IContainerContext storage api return type still has undefined in 0.39 package version.
                 // So once we release 0.40 container-defn package we can remove this check.
                 assert(context.storage !== undefined, 0x1f4 /* "Attached state should have storage" */);
-                const aggrStorage = BlobAggregationStorage.wrap(context.storage, logger);
+                const aggrStorage = BlobAggregationStorage.wrap(context.storage, logger, undefined, packingLevel);
```

Comment:
Add /* allowPacking */ after undefined. See other places where we use /* */ after parameters.